qt/scientific_interfaces/Indirect/AbsorptionCorrections.ui

```diff
@@ -158,6 +158,62 @@
             </property>
            </widget>
           </item>
+          <item row="3" column="0">
+           <widget class="QCheckBox" name="cbSparseInstrument">
+            <property name="text">
+             <string>Sparse Instrument</string>
+            </property>
+            <property name="checked">
+             <bool>false</bool>
+            </property>
+           </widget>
+          </item>
+          <item row="4" column="0">
+           <widget class="QLabel" name="lbNumberOfDetectorRows">
+            <property name="enabled">
+             <bool>false</bool>
+            </property>
+            <property name="text">
+             <string>Number Of Detector Rows:</string>
+            </property>
+           </widget>
+          </item>
+          <item row="4" column="1">
+           <widget class="QSpinBox" name="spNumberDetectorRows">
+            <property name="enabled">
+             <bool>false</bool>
+            </property>
+            <property name="minimum">
+             <number>3</number>
+            </property>
+            <property name="value">
+             <number>3</number>
+            </property>
+           </widget>
+          </item>
+          <item row="4" column="2">
+           <widget class="QLabel" name="lbNumberDetectorColumns">
+            <property name="enabled">
+             <bool>false</bool>
+            </property>
+            <property name="text">
+             <string>Number Of Detector Columns:</string>
+            </property>
+           </widget>
+          </item>
+          <item row="4" column="3">
```

Comment:
Minor point - I think the columns spinbox should be aligned with the "Max Scatter Point Attempts" above it. You can achieve this by changing the column to "4" here:

![image](https://user-images.githubusercontent.com/56295817/210373278-a38feda0-4fdf-400a-847d-cadbd54e9af0.png)

Reply:
Ok, will fix this.